src/app/(pages)/profile/edit/_components/profile-form.tsx

```diff
@@ -49,14 +50,15 @@ interface ProfilePageProps {
 export default function ProfileForm({ userInfo }: ProfilePageProps) {
     const queryClient = useQueryClient()
     const router = useRouter()
+    const { user } = usePrivy()
```

Comment:
privy methods shouldn't be used without checking its "ready" unless we are sure of that.